model/status.go

```diff
@@ -27,6 +27,23 @@ type Status struct {
 	ActiveChannel  string `json:"active_channel,omitempty" db:"-"`
 	DNDEndTime     int64  `json:"dnd_end_time"`
 	PrevStatus     string `json:"-"`
+	MondayStart    string `json:"monday_start"`
```

Comment:
This is making status table schema cumbersome. Maybe you could have seperate model for notification schedule.

Reply:
Also, how is timezone of user handled?

Reply:
Thanks for the first feedback @darkLord19 ! We'll try on working on that. seperating model for notification schedule. And right now, we handle timezone by getting the client's real time.